contentcuration/contentcuration/static/js/edit_channel/share/views.js

```diff
@@ -34,7 +34,7 @@ var MESSAGES = {
     "already_invited": "This person has already been invited.",
     "sending_invitation": "Sending invitation...",
     "invite_failed": "Failed to send invitation",
-    "invite_sent": "Invitation Sent!",
+    "sent": "Sent!",
```

Comment:
Note: on translations, I've been renaming some messages as there are legacy translations for the old ones, so the text won't get updated automatically